doc/content/reference/application-packages/_index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Application Packages"
+description: ""
+weight: 4
+---
+
+This is the reference to work with Application Server application packages.
+
+<!--more-->
+
+Application packages specify state machines running both on the end-device and the Application Server as well as signaling messages exchanged between the end-device's application layer and the Application Server.
+
+{{< cli-only >}}
+
+## Listing the Available Packages
+
+The application packages available for a given device `dev1` of application `app1` can be obtained as follows:
+
+```bash
+$ ttn-lw-cli app pkg ls app1 dev1
```

Comment:
Use full names of commands in documentation (so `applications packages list` in this case). Same below.